README.md

```diff
@@ -1,19 +1,18 @@
 # Concourse pool boy
 
-Like a an efficient pool boy, keep the pools of the [Concourse pool-resource] clean from debris.
-
-For an even smarter pool boy, you can tell him to check if a floating body is still alive before
-cleaning it by using our improved [Pix4D pool-resource] and setting the parameters
-`CONCOURSE_BASE_URL`, `CONCOURSE_USERNAME` and `CONCOURSE_PASSWORD` (see below).
+Observes the pools used by the *Concourse pool resource* and puts back to `unclaimed` any **stale** lock.
 
 Can be used from the command-line or from a Concourse pipeline.
 
+We strongly suggest to use our [forked version of the Concourse pool-resource], since this enables the Pool Boy to query the ATC and determine if a lock is stale based on correct liveliness information about the job which acquired it. To enable the Pool Boy to query the ATC, see section "Enabling ATC querying".
+
+On the other hand, if paired with the original Concourse pool resource, the only criteria it can use for staleness is a timeout, so it is possible for the Pool Boy to steal a lock out of a perfectly fine job and cause mayhem when the job that owned the lock attempts to unclaim it.
```

Comment:
```suggestion
On the other hand, if paired with the original Concourse pool resource, the only criterion it can use for staleness is a timeout, so it is possible for the Pool Boy to steal a lock out of a perfectly fine job and cause mayhem when the job that owned the lock attempts to unclaim it.
```